docs/whats-new/dotnet-aspire-9-release-candidate-1.md

```diff
@@ -532,10 +532,20 @@ Redis resources and they'll each be visible on the Redis Insight dashboard.
 
 ### OpenAI
```

Comment:
Do we want to say `(Preview)` on this? Like we do for Azure Functions.

Reply:
Then I will create another PR